Calling/Controllers/CallRecordingController.cs

```diff
@@ -23,8 +24,9 @@ public class CallRecordingController : Controller
         private const string CallRecodingActiveErrorCode = "8553";
         private const string CallRecodingActiveError = "Recording is already in progress, one recording can be active at one time.";
         public ILogger<CallRecordingController> Logger { get; }
-        static Dictionary<string, RecordingInfo> recordingData = new Dictionary<string, RecordingInfo>();
+        static Dictionary<string, RecInfo> recordingData = new Dictionary<string, RecInfo>();
```

Comment:
I don't think I noticed this before but I think we need to be more explicit about this data store?

How should customers be dealing with this recordingData store in production?
@bharat-kalyan-namburi

Reply:
I would think there should be some type of in memory store that is initialized when the .net server is started up
This would allow developers to identify that this is an in-memory store to maintain their recordings and they could replace this with a redis cache or something larger if necessary.

We don't have to make this sample be something they could build off of for production but we want to share if you were to do something similar this is where you can scale.

Reply:
Moved cache operations into separate singleton class